portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 Google LLC
+//
```

Comment:
The "portfolio" directory doesn't make much sense here.  Just move src to the root directory